src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**RBAC**
+
+* [BREAKING CHANGE] Fix #11883: `az role assignment create`: empty scope will prompt error
```

Comment:
Please place this entry in the right place in the history file following alphabetic order.

Reply:
Thanks, updated.